forked-daapd.conf.in

```diff
@@ -259,6 +259,11 @@ mpd {
 	# the playlist like MPD does. Note that some dacp clients do not show
 	# the playqueue if playback is stopped.
 #	clear_queue_on_stop_disable = false
+
+	# A directory in one of the library directories that will be used as the default
+	# playlist directory. forked-dapd creates new playlists in this directory if only
+	# a playlist name is provided by the mpd client (instead of a full virtual path)
+#	default_playlist_directory = "/srv/music/playlists"
```

Comment:
Generally, if the setting is commented out, and there is a value, it means that this is the value forked-daapd will use unless otherwise configured. That is not the case here. My suggestion is to have an empty string here, and that the functionality is disabled by default (also serves as a security measure). Users who want this must then enter a directory.